src/core/hotkey_config.c

```diff
@@ -26,6 +26,22 @@ static const char *ini_keys[] = {
     "decrease_game_speed",
     "rotate_map_left",
     "rotate_map_right",
+    "build_vacant_house",
+    "build_clear_land",
+    "build_road",
+    "build_plaza",
+    "build_gardens",
+    "build_prefecture",
+    "build_engineers_post",
+    "build_doctor",
+    "build_granary",
+    "build_warehouse",
+    "build_market",
+    "build_wall",
+    "build_gatehouse",
+    "build_reservoir",
+    "build_aquaduct",
```

Comment:
In English it's aqu**e**duct:
```suggestion
    "build_aqueduct",
```